### DISCUSSION TITLE
Rustdesk on Ubuntu LXD container?

### DISCUSSION THREAD

**Discussion by AndnixSH** [rustdesk/rustdesk#3745] [Q&A]
When trying to connect to my LXD container, it says "Unsupported display server type". In the console, it says `(rustdesk:1508): Gtk-WARNING **: 17:42:19.177: cannot open display: ` when I enter `rustdesk`

I have x11 and wayland and some desktops installed but I'm not sure how to open the display for Rustdesk. It works fine if I use xrdp or x2go, but I prefer Rustdesk and I want to be able to remote it outside my home

Is there a way to get Rustdesk to work under LXD container?

I'm using QNAP nas server. The container is running Ubuntu 22.04 and it has access to iGPU PCI `00:02.0 VGA compatible controller: Intel Corporation JasperLake [UHD Graphics] (rev 01)`

**Edit:**

So I came up a briliant idea. Since I had used Google Cloud before, I try to follow excat steps to install desktop enviornment on my LXD https://ubuntu.com/blog/launch-ubuntu-desktop-on-google-cloud

Rustdesk works fine
![image](https://user-images.githubusercontent.com/40742924/227798010-5e354727-2c3b-4c05-a6d9-9d75000f39bb.png)

But the issue is the desktop GUI disappear when closing Google Remote Desktop

![image](https://user-images.githubusercontent.com/40742924/227798629-4ad41f69-4315-4bac-a98e-37f26b721674.png)

Is there a way to set Rustdesk to use Gnome instead Google Remote Desktop?

**Comment by grummbeer** [comment]
https://github.com/rustdesk/rustdesk/issues/59
https://github.com/rustdesk/rustdesk/issues/3592
